lib/config.js

```diff
@@ -24,25 +31,31 @@ var getSysConfRoot = function() {
 
 };
 
-/*
- * Determine whether we are in a browser or cli
+/**
```

Comment:
@jmdrawneek ditto `/**` here. I think we only care about documenting/testing exported functions for now so i'd just leave "private" function alone for now. I'm guessing we will figure out exactly what we want to do with them down the road.